ios/engine/KMEI/KeymanEngine/Classes/Extension/Date+ISOExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Date+ISOExtension.swift
+//  KeymanEngine
+//
+//  Created by Randy Boring on 4/27/19.
+//  Copyright © 2019 SIL International. All rights reserved.
+//
+
+import Foundation
+
+extension Formatter {
+  static let iso8601: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.calendar = Calendar(identifier: .iso8601)
+    formatter.locale = Locale(identifier: "en_US_POSIX")
+    formatter.timeZone = TimeZone(secondsFromGMT: 0)
+    formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
+    return formatter
+  }()
+  static let iso8601noFS: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.calendar = Calendar(identifier: .iso8601)
+    formatter.locale = Locale(identifier: "en_US_POSIX")
+    formatter.timeZone = TimeZone(secondsFromGMT: 0)
+    formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssXXXXX"
+    return formatter
+  }()
```

Comment:
Gabriel had a set of ISO8601 formats defined in
`ios/engine/KMEA/KeymanEngine/Classes/Extension/JSONDecoder.DateDecodingStrategy+ISO8601Fallback.swift`.

These can probably get added there.

Reply:
I would have, but those are extensions of JSONDecoder, whereas these are extensions  of Formatter.
It  is unfortunate that I called the  file Date+ISOExtension instead of Formatter+ISOExtension or DateFormatter+ISOExtension. We could rename  the file...

Reply:
+1 on rename

Reply:
renamed to Formatter+ISODateExtension.swift